lang/en/docs/workspaces.md

```diff
@@ -0,0 +1,96 @@
+---
+id: docs_workspaces
+guide: docs_workspaces
+layout: guide
+---
+
+{% include vars.html %}
+
+Workspaces are a new way to setup your package architecture that's available by default starting from Yarn 1.0. It allows you to setup multiple packages in such a way that you only need to run `yarn install` once to install all of them in a single pass.
+
+### Why would you want to do this?
+
+- Your dependencies can be linked together, which means that your workspaces can depend on one another while always using the most up-to-date code available. This is also a better mechanism than `yarn link` since it only affects your workspace tree rather than your whole system.
+
+- All your project dependencies will be installed together, giving Yarn more latitude to better optimize them.
+
+- Yarn will use a single lockfile rather than a different one for each project, which means less conflicts and easier reviews.
+
+### How to use it?
+
+Add the following in a `package.json` file. Starting from now on, we'll call this directory the "workspace root":
+
+**package.json**
+
+```json
+{
+    "private": true,
+    "workspaces": [
+        "workspace-a",
+        "workspace-b"
+    ]
+}
+```
+
+Then create two subfolders named `workspace-a` and `workspace-b`. In each of them, create another `package.json` file with the following content:
+
+**workspace-a/package.json:**
+
+```json
+{
+    "name": "workspace-a",
+    "version": "1.0.0",
+
+    "dependencies": {
+        "cross-env": "5.0.5"
+    }
+}
+```
+
+**workspace-b/package.json:**
+
+```json
+{
+    "name": "workspace-b",
+    "version": "1.0.0",
+
+    "dependencies": {
+        "cross-env": "5.0.5",
+        "workspace-a": "1.0.0"
+    }
+}
+```
+
+Finally, run `yarn install` somewhere, ideally inside the workspace root. If everything works well, you should now have a similar file hierarchy:
+
+```
+/package.json
+/yarn.lock
+
+/node_modules
+/node_modules/cross-env
+/node_modules/workspace-a -> /workspace-a
+
+/workspace-a/package.json
+/workspace-b/package.json
+```
+
+And that's it! Requiring `workspace-a` from a file located in `workspace-b` will now use the exact code currently located inside your project rather than what is published on Github, and the `cross-env` package has been correctly deduped and put at the root of your project to be used by both `workspace-a` and `workspace-b`.
+
+### How does it compare to Lerna?
+
+Yarn's workspaces are the low-level primitives that tools like Lerna can (and [do](https://github.com/lerna/lerna/pull/899)!) use. They will never try to support the high-level feature that Lerna offers, but by implementing the core logic of the resolution and linking steps inside Yarn itself we hope to enable new usages and improve performances.
+
+### Tips & Tricks
```

Comment:
Just remembered: it is important to mention the following too:

 - Workspaces only works in private packages
 - You can disable workspaces via setting `workspaces-experimental` to `false` in `.yarnrc`